### ISSUE TITLE
There is a calibration issue which is new since 1.06

### ISSUE THREAD

**Issue by BarbourSmith** [MaslowCNC/Firmware#398]
Running the attached file results in a curved line instead of a straight one. The issue is solved by reverting to firmware 1.03 so we know the issue is new.

[ttrapp2.zip](https://github.com/MaslowCNC/Firmware/files/1742502/ttrapp2.zip)

The original issue and information can be found in this forum thread:

https://forums.maslowcnc.com/t/velocity-slow-down-and-arced-cut/2413/48

I'm a little stumped on this one so it might be a good candidate for git-bisect to start narrowing down what is going on

**Comment by krkeegan** [comment]
I read through the forum, has anyone else other than clintloggins been able to reproduce this error?  @BarbourSmith have you been able to reproduce this error?

**Comment by BarbourSmith** [comment]
I believe that @blurfl was able to recreate it. I've been bogged down in updating the calibration process which has meant that my machine has not been a good test platform since it's calibration has been off. I'm hoping to wrap that up tomorrow and be able to test then.

**Comment by GeroBH** [comment]
I cut it The outline only) today with 1.07. No curve visable
https://discourse-cdn-sjc2.com/standard11/uploads/maslowcnc/optimized/2X/8/84a8b20a28af176793f6704b7e08f9d925298905_1_282x500.jpg

**Comment by BarbourSmith** [comment]
That is fantastic! 

It makes me nervous that we don't know what was causing the issue or what made it go away

**Comment by reecej** [comment]
I agree. This has been a strange one, but I'm glad it is working better. I don't see anything in the changes for v1.07 that would jump out at me as resolving this.
